plugins/CoreUpdater/tests/Integration/ReleaseChannelTest.php

```diff
@@ -57,12 +57,12 @@ public function test_getUrlToCheckForLatestAvailableVersion()
     {
         $version = Version::VERSION;
         $phpVersion = urlencode(PHP_VERSION);
-        $mysqlVersion = Db::get()->getServerVersion();
+        $mysqlVersion = urlencode(Db::get()->getServerVersion());
         $url = urlencode(Url::getCurrentUrlWithoutQueryString());
 
-        $urlToCheck = $this->channel->getUrlToCheckForLatestAvailableVersion();
+        $urlToCheck = urlencode($this->channel->getUrlToCheckForLatestAvailableVersion());
 
-        $this->assertStringStartsWith("http://api.matomo.org/1.0/getLatestVersion/?piwik_version=$version&php_version=$phpVersion&mysql_version=$mysqlVersion&release_channel=my_channel&url=$url&trigger=&timezone=", $urlToCheck);
+        $this->assertStringStartsWith(urlencode("http://api.matomo.org/1.0/getLatestVersion/?piwik_version=$version&php_version=$phpVersion&mysql_version=$mysqlVersion&release_channel=my_channel&url=$url&trigger=&timezone="), $urlToCheck);
```

Comment:
Why are those changes needed? Just wondering as those tests were passing locally. 🤔

Reply:
right I think the DB I used MariaDB, has some special characters